### PR TITLE
WIP: Cannot get coverage quite there

### PR DESCRIPTION
**Pull request by andramalmen**

### CODE REVIEW

File: src/divide.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+/**
+ * @module Divide
+ * @Description This method will perform Divide operation.
+ *
+ * @param {Number} div1 - the number that is divided, divident
+ * @param {Number} div2 - the number to be divided with, divisor
+ * @return {Number} - Result of Division
+ */
+function Divide(div1, div2) {
```
Comment (kentcdodds) [inline src/divide.js]: I'd prefer this to be lower-cased

**Comment by kentcdodds** [comment]
I'm happy to merge this PR, but if you'd like to get some practice, the commit message doesn't follow [our guidelines](https://github.com/eggheadio-github/stack-overflow-copy-paste/blob/master/CONTRIBUTING.md). Would you like to update that?

**Comment by andramalmen** [comment]
All done!

**Comment by kentcdodds** [comment]
Looks good!

**Pull request merged**